packages/mobile/src/harmony-native/components/Text/Text.tsx

```diff
@@ -39,14 +42,15 @@ export const Text = forwardRef<TextBase, TextProps>((props, ref) => {
   const variantStyles = variant && variantStylesMap[variant]
   const t = theme.typography
 
-  const fontWeight = variantStyles?.fontWeight[strength]
+  const fontWeight = strength && variantStyles?.fontWeight[strength]
 
   const textStyles: TextStyle = css({
     ...(variantStyles && {
-      fontSize: t.size[variantStyles.fontSize[size]],
-      lineHeight: t.lineHeight[variantStyles.lineHeight[size]],
-      fontFamily: t.fontByWeight[variantStyles.fontWeight[strength]],
-      ...('css' in variantStyles ? variantStyles.css : {})
+      fontSize: size && t.size[variantStyles.fontSize[size]],
+      lineHeight: size && t.lineHeight[variantStyles.lineHeight[size]],
+      fontFamily:
+        strength && t.fontByWeight[variantStyles.fontWeight[strength]],
+      ...('css' in variantStyles ? variantStyles.css ?? {} : {})
```

Comment:
do we not want to allow these to be overridden by the size prop? or do they not clash?

Reply:
no clash, just for label varient i believe